validator/lib/validatePnts.js

```diff
@@ -181,6 +183,16 @@ function validatePnts(content) {
         return 'Feature table property BATCH_LENGTH must be less than or equal to POINTS_LENGTH.';
     }
 
+    if (defined(featureTableJson.BATCH_ID)) {
+        var featureTable = new Cesium3DTileFeatureTable(featureTableJson, featureTableBinary);
+        var batchIds = featureTable.getPropertyArray('BATCH_ID', ComponentDatatype.UNSIGNED_SHORT, 1);
+        for (var i = 0; i < batchIds.length; i++) {
```

Comment:
As a style suggestion, we usually factor out the array length into its own variable, like:

```
var length = batchIds.length;
for (var i = 0; i < length ; i++) {
    ...
}
```

This is generally good practice in JavaScript for performance intensive areas. Even though this area isn't one of those we still prefer to write for loops in this way.